### ISSUE TITLE
Streets on borders (Chicago/Evanston)

### ISSUE THREAD

**Issue by jidanni** [gravitystorm/openstreetmap-carto#3218]
Actually the north side of the street is just Howard Street,
the south half is West Howard Street.

In fact "W. Howard St." is how it is commonly written too.

https://www.openstreetmap.org/#map=19/42.01933/-87.69661

**Comment by kocio-pl** [comment]
I don't think it's a rendering problem, we just render what is tagged here:

https://www.openstreetmap.org/way/313292998

**Comment by jidanni** [comment]
OK then it is a more systematic problem, as I see both
tiger:zip_left	60645
tiger:zip_right	60202

**Comment by kocio-pl** [comment]
We render only `name=*` tag, not some other tags and numbers.

**Comment by jidanni** [comment]
Yes I am saying that the program they are using to insert the data has the zip codes for both sides, but the street name for only one side. So please reassign this bug to that team. Thanks.

**Comment by SomeoneElseOSM** [comment]
(not a rendering issue, but) the normal way to tag such things is with e.g. "name:left" and "name:right".  See https://taginfo.openstreetmap.org/keys/name%3Aleft#overview (25k objects already tagged that way) and https://wiki.openstreetmap.org/wiki/Names#Left_and_right_names .  Some renderers do render name:left and name:right.

**Comment by jidanni** [comment]
OK I left a comment on https://www.openstreetmap.org/changeset/57802738
